### PR TITLE
Basic users

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Created layout and updating of user selection

### CODE REVIEW

File: elements/timeline-picker.html [hunk 1]
```diff
@@ -42,24 +44,32 @@
...
                 id="ajax-timeline-data"></iron-ajax>
 
-        <iron-ajax
-            id="pickedTimelinesPost"
-            url="/timeline/picked-timelines"
-            handle-as="json"
-            method = "POST"
-            content-type="application/json"
-            on-response="handlePickedTimelinesResponse"></iron-ajax>
+        <!--<iron-ajax-->
```
Comment (geenen124) [inline elements/timeline-picker.html]: Please remove commented code or provide a reason (e.g. a TODO)

File: elements/timeline-picker.js [hunk 2]
```diff
@@ -1,37 +1,67 @@
 class timelinePicker {
+
+
```
Comment (geenen124) [inline elements/timeline-picker.js]: Unnecessary whitespace

File: objects/ProjectManager.js [hunk 3]
```diff
@@ -115,6 +115,10 @@ class ProjectManager {
         });
     }
 
+    getUsers(XMLObject) {
+
```
Comment (geenen124) [inline objects/ProjectManager.js]: Please add a TODO or remove this method

**Comment by Markvanderuit** [comment]
Looks cool, and seems to work as expected. We'll talk about colors/layout later.

Merging.
